tests/plugin_http_api_test.py

```diff
@@ -1025,23 +1025,8 @@ def test_ProducerApi(self) :
         # create_snapshot with empty parameter
         default_cmd = cmd_base + "create_snapshot"
         ret_json = Utils.runCmdReturnJson(default_cmd)
-        self.assertEqual(ret_json["code"], 500)
```

Comment:
I'm not too familiar with these tests, why delete all of these?

Reply:
Snapshot generation was disabled, so after reenabling the feature this error code is no longer valid.